lib/segment/analytics/defaults.rb

```diff
@@ -6,15 +6,22 @@ module Request
         PORT = 443
         PATH = '/v1/import'
         SSL = true
-        HEADERS = { :accept => 'application/json' }
```

Comment:
was this previously unused?

Reply:
Yes. It was being assigned to a variable, but the actual headers used were hardcoded. 

(The diff for [this individual commit](https://github.com/segmentio/analytics-ruby/pull/134/commits/954a2c91dd8b893a786fdf55f5db500846d14f7c) includes all the relevant files).